flocker/node/agents/test/test_ebs.py

```diff
@@ -246,3 +249,34 @@ def test_no_new_device_logged(self, logger):
                 time_limit=0,
             )
         )
+
+
+class FindAllocatedDeviceTests(TestCase):
+    """
+    Tests for finding allocated devices.
+    """
+
+    def test_returns_device_name_list(self):
+        """
+        Returns at least one device (the root device).
+        """
+        self.assertGreater(len(_find_allocated_devices()), 0)
```

Comment:
And maybe also that it starts with /dev/? Or that they exist and are block devices?